deliveries/chainide-polkaholic-integrated-wasm-contract-explorer-integration-milestone_1.md

```diff
@@ -0,0 +1,190 @@
+    - Integrated WASM Developer dashboard (as an iframe), or direct link to the dashboard using the developer's wallet address; this can be used to link any developer dashboard. 07:26
+    - Ask whether developers wish to (a) share the ABI/metadata (b) have their code verified and submitted for open source. 13:04
+    - Use cases to post user's contract-related data. 13:15
+    - A link to the contract call will take users to Polkaholic.io using the extrinsicHash generated. 15:15
+
+Using EVM:
+
+- A pre-built Linux container image includes a set of Hardhat, Truffle, and Node.js in ChainIDE-sandbox at the backend. 29:32
+- An Astar EVM dashboard in the front end:
+    - A wallet panel that allows users to connect MetaMask wallet in the browser switch to the mainnet, testnet, and devnet; mint testnet & devnet coin. 30:04
+    - Compilation of Substrate smart contracts, including compilation support for Solidity. 30:48
+    - Select the compiled contract to deploy to Astar Network. 31:13
+    - Interact with the deployed contracts. 31:47
+    - Monitor Astar Network backend real-time activities. 32:05
+    - Includes some useful plugins: JavaScript Virtual Machine Wallet (JSVM), Flattener, & Debugger. 32:48
+
+### Network Support: from Milestone 1 to Milestone 2
+
+This project was completed and tested mostly on Shibuya but the following chains are supported by both teams:
+
+| {network} | Relay Chain / Para ID | Integration |  
+|----|---|---|
+| `astar` | Polkadot / 2006 | Milestone 1 |
+| `shiden` | Kusama / 2007 |  Milestone 1 |
+| `shibuya` | Testnet | Milestone 1 |
+| `rococo-contracts` | Rococo / 1002 | Milestone 2 possible (Note: Parity support required due to low numbers of peers, syncing halted) |
+
+ChainIDE contains the ability to add arbitrary chains, while Polkaholic.io is able to add additional chains upon request in Milestone 2 based on community feedback.
+
+### Current Status / Future Plans
+
+Both teams will complete Milestone 1 in September and are developing plans to propose Milestone 2 in November, after using October as a feedback collection period.  
+
+Here is the detailed timeline:
+* September 2023 - ChainIDE is awaiting QA from Colorful Notion while Astar is currently undergoing acceptance testing.  Colorful Notion is completing this document and finalizing functionality for a developer preview in October 2023.
+
+* *October 2023* - ChainIDE + Colorful Notion  request feedback from curators, all inkubator WASM Contract Developers and experienced WASM Contract Developers to:
+
+	1. Try ChainIDE at [https://staging-9589904a8a.chainide.com/s/dashboard/projects](https://staging-9589904a8a.chainide.com/s/dashboard/projects) using Shibuya and the SBY faucet.   We recommend using PSP22, PSP34 or the UniswapV2 contract templates, and then using your own WASM Contract.  
+
+	2. For bug reports and feature requests, submit them [here](TODO) or join our open group here [ChainIDE+Colorful Notion+Astar]()  
```

Comment:
There's a TODO linked here

Reply:
We have adjusted it to point to the inkubator matrix room.  If you have a better idea, pls advise.